Meta/Lagom/CMakeLists.txt

```diff
@@ -602,6 +601,9 @@ if (BUILD_LAGOM)
             add_serenity_subdirectory("Tests/${dir}")
         endforeach()
 
+        # LibTLS needs a special working directory to find cacert.pem
+        lagom_test(../../Tests/LibTLS/TestTLSHandshake.cpp LibTLS LIBS LibTLS LibCrypto)
```

Comment:
But no working directory is set here.

Reply:
It sets `CMAKE_CURRENT_BINARY_DIR` by default. The tests above where it was removed use `CMAKE_CURRENT_SOURCE_DIR`.

Reply:
Ah. Alright.